examples/mobile/HomeApp/js/app.js

```diff
@@ -171,7 +218,16 @@ import Storage from "./clipping-storage.js";
 		burgerButton.addEventListener("click", onButtonClick);
 		popupButton.addEventListener("click", onPopupSubmit);
 
-		loadWebClipList();
+		appsList = storage.readAllFromStorage(Storage.elements.APPSLIST);
```

Comment:
You can add default value for `appsList` eg.;
```
appsList = storage.readAllFromStorage(Storage.elements.APPSLIST) || [];
```

Reply:
done